### PR TITLE
Remove implicit dependency on rake gem

### PR DESCRIPTION
**Pull request by CAMOBAP**

### CODE REVIEW

File: lib/extract_ttc.rb [hunk 1]
```diff
@@ -7,16 +7,23 @@
...
   extend FFI::Library
   # NOTE: ffi doesn't support bundles out of box https://github.com/ffi/ffi/issues/42#issuecomment-750031554
   # NOTE: rake-compiler doesn't support dylib generation https://github.com/rake-compiler/rake-compiler/issues/183
-  lib_name = Rake::ExtensionTask.new("stripttc", Gem::Specification::load("../extract_ttc.gemspec"))
-    .binary(RUBY_PLATFORM)
+  lib_name = if File.exists?(File.join(File.dirname(__FILE__), "stripttc.bundle"))
+               "stripttc.bundle"
+             else
+               "stripttc.so"
+             end
```
Comment (alexeymorozov) [inline lib/extract_ttc.rb:26]: 👍

File: lib/extract_ttc.rb [hunk 2]
```diff
@@ -7,16 +7,23 @@
```
Comment (alexeymorozov) [inline lib/extract_ttc.rb:7]: This PR handles some runtime dependency issue? Do we need to remove the "require" line?

**Comment by alexeymorozov** [comment]
LGTM! Please merge any time :)

File: lib/extract_ttc.rb (hunk 2 above)
Comment (CAMOBAP) [reply lib/extract_ttc.rb:7]: good catch those 2 lines above need to be removed

**Comment by ronaldtse** [comment]
Thanks @CAMOBAP @alexeymorozov !

**Pull request merged**